app/code/Magento/CatalogImportExport/Model/Import/Product/Validator.php

```diff
@@ -227,7 +227,12 @@ public function isAttributeValid($attrCode, array $attrParams, array $rowData)
                 $valid = $this->validateOption($attrCode, $attrParams['options'], $rowData[$attrCode]);
                 break;
             case 'multiselect':
-                $values = $this->context->parseMultiselectValues($rowData[$attrCode]);
+                $separator = $this->context->getMultipleValueSeparator();
+                // added to prevent backward compatibility since before we didn't use separator param value
+                $values = $separator === ','
```

Comment:
Let's replace this `','` with a private constant, probably call it `DEFAULT_GLOBAL_MULTI_VALUE_SEPARATOR`, to easier readability.